### PR TITLE
Issue #10545: Add support for messages in inlined config

### LINKED ISSUE #10545: Add support for messages in inlined config
detected at https://github.com/checkstyle/checkstyle/pull/10532#discussion_r681905396

### PR DESCRIPTION
**Pull request by shashwatj07**
Resolves #10545

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/bdd/TestInputConfiguration.java [hunk 1]
```diff
@@ -38,16 +38,21 @@
     /** Map of non default properties. */
     private final Map<String, String> nonDefaultProperties;
 
+    /** Map of non default properties. */
```
Comment (timurt) [inline src/test/java/com/puppycrawl/tools/checkstyle/bdd/TestInputConfiguration.java:41]: correct javadoc

File: src/test/java/com/puppycrawl/tools/checkstyle/bdd/TestInputConfiguration.java [hunk 2]
```diff
@@ -98,12 +110,16 @@ public void setCheckName(String checkName) {
...
+            defaultProperties.put(propertyName, defaultPropertyValue);
+        }
+
+        public void addNonDefaultProperty(String propertyName, String nonDefaultPropertyValue) {
+            nonDefaultProperties.put(propertyName, nonDefaultPropertyValue);
         }
 
-        public void addNonDefaultProperty(String key, String value) {
-            nonDefaultProperties.put(key, value);
+        public void addMessage(String messageKey, String messageString) {
```
Comment (timurt) [inline src/test/java/com/puppycrawl/tools/checkstyle/bdd/TestInputConfiguration.java:121]: lets call it `addCheckMessage`

**Review by timurt — CHANGES_REQUESTED** [review]

File: src/test/java/com/puppycrawl/tools/checkstyle/bdd/TestInputConfiguration.java (hunk 1 above)
Comment (shashwatj07) [reply src/test/java/com/puppycrawl/tools/checkstyle/bdd/TestInputConfiguration.java:41]: done

File: src/test/java/com/puppycrawl/tools/checkstyle/bdd/TestInputConfiguration.java (hunk 2 above)
Comment (shashwatj07) [reply src/test/java/com/puppycrawl/tools/checkstyle/bdd/TestInputConfiguration.java]: done

**Review by romani — APPROVED** [review]

**Pull request merged**